src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/Pages/Todo.razor.cs

```diff
@@ -9,29 +9,24 @@ public partial class Todo
 
     public bool IsBussy { get; set; } = false;
     TodoItemDto todoItem = new();
```

Comment:
I think this variable needs a name change (from `todoItem` to `TodoItem`) since in the .razor file this change is already applied.